proto/buf.yaml

```diff
@@ -1,7 +1,7 @@
 version: v1
 name: buf.build/cosmos/ibc
 deps:
-  - buf.build/cosmos/cosmos-sdk:v0.47.0
+  - buf.build/cosmos/cosmos-sdk:9d547dbea90f47afbe1898388fcebffb
```

Comment:
Looks like latest commit/tag on buf is https://buf.build/cosmos/cosmos-sdk/docs/aa25660f4ff746388669ce36b3778442

I assume the guys will make a v0.50.0 tag when they're ready